### ISSUE TITLE
Make "Getting Started" more prominent

### ISSUE THREAD

**Issue by SimonBiggs** [pymedphys/pymedphys#1538]
Hi @Matthew-Jennings,

Would we be able to make the overview and getting started documents be right at the top of the navigation:

![image](https://user-images.githubusercontent.com/6559099/127405152-95701f6f-1916-4296-8f75-58122f4be815.png)

Is that something you'd be happy to make happen?

**Comment by SimonBiggs** [comment]
Mike ended up finding himself on the developer getting started page when he was just trying to use the library:

https://pymedphys.discourse.group/t/windows-setup/128/2?u=simonbiggs

It might also be worth having a note at the top of the contributor guides using a big "Note" box that says something like, "If all you want to do is use PyMedPhys, you're in the wrong place, go to --getting started--" instead.

Also... There hasn't been much uptake of the application in the general public. I wonder if PyMedPhys should consider leaving the GUI as an undocumented feature. Right now it is taking up valuable real-estate in the docs, but the docs don't actually provide much (any) guidance on how to use it.

**Comment by sjswerdloff** [comment]
I am of the opinion that the (application level) GUI belongs in a separate project/repo.
If it weren't for the heavy use of the CLI, I'd recommend the same for that.
That would leave the library as one project and applications in separate projects (perhaps one "GUI Apps", perhaps separate projects for separate GUIs).  There is GUI that is at the library level (shared amongst apps), so I could see that staying in the library project.

**Comment by SimonBiggs** [comment]
> I am of the opinion that the (application level) GUI belongs in a separate project/repo.

Main issue with separate repo, is chaining together CI. It's a mighty pain.

**Comment by Matthew-Jennings** [comment]
What about separating the documentation entirely? One site for _Library_ and another for _App_. It seems to me that CLI docs can probably fit nicely enough in library. Then the pymedphys landing page can point to each?

This helps solve the "real-estate" problem without implicitly prioritising the library over the app. I take your point that the app is hardly used, but that may change if/when it's built out more. Having a space for it to build out might encourage others more interested in app development to dive in too?

Note boxes like the one you described sound great to me!

I can't really comment on the relative pros/cons of separating library and apps into different repos...

**Comment by SimonBiggs** [comment]
> What about separating the documentation entirely? One site for _Library_ and another for _App_. It seems to me that CLI docs can probably fit nicely enough in library. Then the pymedphys landing page can point to each?

Yup, I like that idea. I suspect it hits the "best of both worlds". ...so ... who feels like designing a landing page? Any volunteers? :)

**Comment by SimonBiggs** [comment]
> What about separating the documentation entirely? One site for _Library_ and another for _App_. It seems to me that CLI docs can probably fit nicely enough in library. Then the pymedphys landing page can point to each?

How about, as a stepping stone, I pull out the app from the docs, and make the current docs be the "library + cli" version of this. And then, when someone is keen then they can make the landing-page + app component.

**Comment by SimonBiggs** [comment]
> I am of the opinion that the (application level) GUI belongs in a separate project/repo.

@sjswerdloff a question about this request, is one of your main motivations so that installing PyMedPhys with the recommended installation `pip install` doesn't bring in all of the GUI dependencies? This would make the default recommended install work on more systems (such as ARM) and be generally lighter.

If that is one of your main motivations, can I propose an alternative approach. What if the main installation method was to run something like:

```bash
pip install pymedphys[lib,cli]
```

The user can opt to leave out the cli flag, or leave out the lib flag if they only want one or the other. The user could also opt to run:

```bash
pip install pymedphys[gui]
```

If they want the GUI dependencies.

...this could make it so that those who want the GUI can pull in the dependencies for it optionally. But, the default recommended install ends up being much lighter.

@sjswerdloff what do you think?
